### ISSUE TITLE
Plugin not working

### ISSUE THREAD

**Issue by phuongnl15395** [HugoHasenbein/redmine_more_previews#51]
after installing plugin I'm not seeing any change, still can't preview the file (Libre is installed on server).
I'm using Redmine 4.0.0.stable, centos 7, LibreOffice 7.3.2.2.

**Comment by phuongnl15395** [comment]
yes, i'm already config it but when i try file with *.docx nothing happend

**Comment by phuongnl15395** [comment]
@HugoHasenbein  could you help me with this?

**Comment by HugoHasenbein** [comment]
whats wrong?

**Comment by phuongnl15395** [comment]
Like I mention above, seems like the plugin not working, can you help me with debuging?

**Comment by HugoHasenbein** [comment]
yes, please  share what you are seeing (or not seeing) with  this plugin.

**Comment by phuongnl15395** [comment]
- Are there any other channel I can chat direct with you?

**Comment by HugoHasenbein** [comment]
use email: ***@***.***

**Comment by phuongnl15395** [comment]
> 

I'm not able to see it, sorry I'm kinda new to this

**Comment by HugoHasenbein** [comment]
did you copy the plugin in the plugins folder and did you call 'bundle install' ? then in redmine go to Administration->Plugins->Redmine More Previews and activate those file types you want to preview

**Comment by phuongnl15395** [comment]
Yes, I'm already done all that step, I can see it in Administration -> plugin -> active Libre. But the problem after done those step, I upload file but I can preview it

**Comment by HugoHasenbein** [comment]
and you activated the corresponding file type f.i. docx for Microsoft Word files? Further, in the plugin configuration there is a green hook mark behind LibreOffice available?

**Comment by phuongnl15395** [comment]
- and you activated the corresponding file type f.i. docx for Microsoft Word file <- Yes
- green hook mark behind LibreOffice available? <- Yes
- Is there anything else that I should check?

**Comment by HugoHasenbein** [comment]
turn on debug, you should see something likes the attached image.
Which version of redmine are you using? 4.2.5? 5.0.0?

**Comment by phuongnl15395** [comment]
Currently, I'm using Redmine version 4.0.0
"turn on debug, you should see something likes the attached image.",
> I already turn on debug, but I dont see anything

**Comment by HugoHasenbein** [comment]
can you send a log file output?

The logfile is located at <Redmine Root>/log/production.log (if you are in production mode)

On a Linux console type

tail -f log/production.log

Then send me the output, which is created, when you click on a file to be previewed.

**Comment by phuongnl15395** [comment]
I give you the log output
Started GET "/attachments/1856" for 103.245.252.76 at 2022-05-04 12:39:18 +0700
Processing by AttachmentsController#show as HTML
  Parameters: {"id"=>"1856"}
  Current user: phuongnl4 (id=30)
  Rendering attachments/other.html.erb within layouts/base
  Rendered common/_no_preview.html.erb (0.3ms)
  Rendered common/_other.html.erb (0.9ms)
  Rendered layouts/_file.html.erb (12.2ms)
  Rendered attachments/other.html.erb within layouts/base (12.9ms)
  Rendered plugins/redmine_more_filters/app/views/hooks/redmine_more_filters/_toggle_operator.html.erb (0.2ms)
  Rendered plugins/redmine_more_filters/app/views/hooks/redmine_more_filters/_css.html.erb (0.3ms)
  Rendered plugins/redmineup_tags/app/views/tags/_additional_assets.html.erb (0.6ms)
  Rendered plugins/additionals/app/views/additionals/_html_head.html.slim (0.6ms)
  Rendered plugins/additionals/app/views/additionals/_body_top.slim (0.2ms)
  Rendered plugins/additionals/app/views/additionals/_content.html.slim (0.2ms)
  Rendered plugins/redmineup_tags/app/views/tags/_select2_transformation_rules.html.erb (0.3ms)
  Rendered plugins/additionals/app/views/additionals/_body_bottom.html.slim (0.3ms)
Completed 200 OK in 59ms (Views: 44.0ms | ActiveRecord: 5.0ms)

**Comment by HugoHasenbein** [comment]
I can see no error. When on the preview page" click on "reload" and see if the log file outputs different information. Else, send the a screen shot of the following pages:

 - Redmine More previews Plugin configuration page (does not reveal sensitive information)
 - Administration -> Info (does not reveal sensitive information)
 - the actual preview page (blur out information, which may be sensitive)

**Comment by phuongnl15395** [comment]
![image](https://user-images.githubusercontent.com/67360660/166628721-3a63e07b-8cee-45b5-b985-2fe8c268a6d2.png)
![image](https://user-images.githubusercontent.com/67360660/166628745-c1616f34-c952-45cf-8cce-3fe73854ee3a.png)
![image](https://user-images.githubusercontent.com/67360660/166628766-b1ca1679-ec95-4f54-b494-58ca788dd866.png)
 I sent you attachment

**Comment by phuongnl15395** [comment]
![image](https://user-images.githubusercontent.com/67360660/166628847-9b97dfae-9062-423b-91e7-f799a310d934.png)

**Comment by HugoHasenbein** [comment]
and below Libre there is a mark like this?

**Comment by phuongnl15395** [comment]
![image](https://user-images.githubusercontent.com/67360660/166629509-480d13e9-5a46-4566-ad6c-f45249462cbe.png)
![image](https://user-images.githubusercontent.com/67360660/166629527-8ae153f7-989b-42b0-a3cc-7995212d2d76.png)
Yes

**Comment by HugoHasenbein** [comment]
Gee, that's a tough one. I want to make the plugin more robust, so good you send me your information.

Just as an experiment: try to rename your document with only ASCII characters, like test.docx and try to preview the test.docx file. I want to make sure, there is no character encoding issue with the filename, when the plugin tries to determine, the file type with mimemagic.

**Comment by phuongnl15395** [comment]
![image](https://user-images.githubusercontent.com/67360660/166630023-f63ce0f0-8dc6-47b8-93a0-0d65e04abed6.png)
I already tried it before. You can see it here, just take another screenshot for you

**Comment by HugoHasenbein** [comment]
I never had this issue before.

would you mind, amending the code according to my instructions to see, what is going on?

in

redmine_more_previews/lib/redmine_more_previews/patches/attachments_controller_patch.rb

go to line 141 and add the following lines of code

141        def show_with_more_previews

     Rails.logger.info @attachment
     Rails.logger.info @attachment.preview_convertible?
     Rails.logger.info RedmineMorePreviews::Converter.mime( @attachment.diskfile )
     Rails.logger.info RedmineMorePreviews::Converter.conversion_mime( @attachment.diskfile )

142          if request.format.html? && @attachment.preview_convertible?
143            render :action => 'more_preview', :formats => :html
144         else
145            show_without_more_previews
146         end
147        end #def

you should get a log file output like

     Rails.logger.info @attachment
     Rails.logger.info @attachment.preview_convertible?
     Rails.logger.info RedmineMorePreviews::Converter.mime( @attachment.diskfile )
     Rails.logger.info RedmineMorePreviews::Converter.conversion_mime( @attachment.diskfile )

you should see something like this

Processing by AttachmentsController#show as HTML
  Parameters: {"id"=>"11876", "filename"=>"test.docx"}
  Current user: admin (id=1)
#<Attachment:0x00007fe0f01857d0>
true
{"active"=>"1", "format"=>"html", "converter"=>"libre", "mime"=>"application/vnd.openxmlformats-officedocument.wordprocessingml.document", "synonyms"=>nil, "icon"=>"docx.png"}
text/html
  Rendering plugins/redmine_more_previews/app/views/attachments/more_preview.html.erb within layouts/base
  Rendered plugins/redmine_more_previews/converters/cliff/app/views/redmine_more_previews/hooks/_cliff_preview_attachment_contextual.html.erb (2.1ms)
  Rendered plugins/redmine_more_previews/converters/cliff/app/views/redmine_more_previews/hooks/_cliff_preview_attachment_top.html.erb (1.8ms)
  Rendered layouts/_file.html.erb (22.9ms)
  Rendered plugins/redmine_more_previews/app/views/attachments/more_preview.html.erb within layouts/base (23.8ms)
  Rendered plugins/redmine_business_case/app/views/hooks/redmine_business_case/_css.html.erb (0.1ms)
  Rendered plugins/redmine_business_case/app/views/hooks/redmine_business_case/_javascripts.html.erb (0.1ms)
  Rendered plugins/redmine_more_filters/app/views/hooks/redmine_more_filters/_toggle_operator.html.erb (0.1ms)
  Rendered plugins/redmine_more_filters/app/views/hooks/redmine_more_filters/_css.html.erb (0.2ms)
  Rendered plugins/redmine_all_thumbnails/app/views/hooks/_html_head.html.erb (0.1ms)
  Rendered plugins/redmine_attachment_categories/app/views/hooks/attachment_categories/_html_head.html.erb (0.1ms)
  Rendered plugins/redmine_issue_attachments/app/views/hooks/issue_attachments/_view_layouts_base_html_head.html.erb (0.1ms)
  Rendered plugins/redmine_business_case/plugins/google_geo_code/app/views/hooks/redmine_business_case/google_geo_code/_css.html.erb (0.2ms)
  Rendered plugins/redmine_more_previews/app/views/hooks/redmine_more_previews/_icon_css.html.erb (2.2ms)
  Rendered plugins/redmine_scripting_engine/app/views/hooks/redmine_scripting_engine/_css.html.erb (0.2ms)
  Rendered plugins/redmine_scripting_engine/app/views/hooks/redmine_scripting_engine/_js.html.erb (0.1ms)
  Rendered plugins/redmine_scripting_engine/app/views/hooks/redmine_scripting_engine/_global_sidebar.html.erb (33.7ms)
Completed 200 OK in 137ms (Views: 93.0ms | ActiveRecord: 9.9ms)
Started GET "/attachments/more_preview/11876/index.html" for 90.187.193.33 at 2022-05-04 06:41:47 +0000
Processing by AttachmentsController#more_preview as HTML
  Parameters: {"id"=>"11876"}
  Current user: admin (id=1)
  Rendering text template
  Rendered text template (0.0ms)
Sent data index.html (1.4ms)
Completed 200 OK in 9057ms (Views: 0.5ms | ActiveRecord: 46.0ms)


if you do see something like this

#<Attachment:0x0000557213c84c38>
true
{"active"=>"1", "format"=>"html", "converter"=>"zippy", "mime"=>"application/zip", "synonyms"=>nil, "icon"=>nil}
text/html

then there is a mimemagic issue. Sometimes mimemagic fails to recognize word docx documents as docx, but it recognizes them as zip files.

**Comment by HugoHasenbein** [comment]
to make things easier: try to preview the attached word-file.

**Comment by phuongnl15395** [comment]
letme try then i will reply back to you

**Comment by phuongnl15395** [comment]
![image](https://user-images.githubusercontent.com/67360660/166640316-4cb600a6-aa32-40bf-a2d8-039f704f89b6.png)
![image](https://user-images.githubusercontent.com/67360660/166640503-1f992cac-9c7d-4bcc-8ae5-1662026f83fa.png)

**Comment by phuongnl15395** [comment]
Am I do it correctly?

**Comment by HugoHasenbein** [comment]
did you restart redmine after the amendment?

**Comment by phuongnl15395** [comment]
Yes I did,

**Comment by HugoHasenbein** [comment]
Ok, this is an issue I had not in the past.

The attachment_controller patch is not executed at all, it seems. Else the log file should have shown some output.

Go to the console

rails c -e production

in there type

AttachmentsController.new.methods.include?(:show_with_more_previews)

it should result with 'true'

**Comment by phuongnl15395** [comment]
![image](https://user-images.githubusercontent.com/67360660/166644506-6cb39281-5111-4e82-b3bf-f06b680c74d7.png)
Its false

**Comment by HugoHasenbein** [comment]
good, actually not good, but this is a first hint. I suspect a plugin collision.

could you move all plugins but this plugin into another folder, let's say to a folder named "deactivated_plugins", which is a sibling of the plugins folder. Restart redmine and try again rails console

**Comment by phuongnl15395** [comment]
I dont think its a good idea because you can see it's currently production env. Any other option?

**Comment by shurakenas** [comment]
I have the same problems with this plugin.

**Comment by HugoHasenbein** [comment]
Then do

AttachmentsController.ancestors

in the console and send me the out put as text

There will be no sensitive information in the output, but I will need more than a few minutes to peruse the information

Thank your for your patience, it helps to make the plugin better

**Comment by phuongnl15395** [comment]
@HugoHasenbein 
2.6.3 :003 > AttachmentsController.ancestors
 => [AttachmentsController, RedmineLightbox2::AttachmentsPatch, #<Module:0x00000000065219c0>, ApplicationController, Additionals::Patches::ApplicationControllerPatch::InstanceMethods, Additionals::Patches::ApplicationControllerPatch, RedmineExtensions::RenderingHelper, RedmineExtensions::ApplicationControllerPatch::InstanceMethods, RedmineExtensions::ApplicationControllerPatch, Redmine::SudoMode::Controller, Redmine::MenuManager::MenuController, Redmine::Search::Controller, RoutesHelper, Redmine::Hook::Helper, Redmine::Pagination, Redmine::I18n, #<Module:0x000000000165a198>, #<Module:0x00000000027fb460>, #<Module:0x00000000027fb500>, ActionController::Base, OpenIdAuthentication, RedmineupTags::Patches::ActionControllerPatch, InvisibleCaptcha::ControllerExt, ActiveRecord::Railties::ControllerRuntime, ActionDispatch::Routing::RouteSet::MountedHelpers, ActionController::ParamsWrapper, ActionController::Instrumentation, ActionController::Rescue, ActionController::HttpAuthentication::Token::ControllerMethods, ActionController::HttpAuthentication::Digest::ControllerMethods, ActionController::HttpAuthentication::Basic::ControllerMethods, ActionController::DataStreaming, ActionController::Streaming, ActionController::ForceSSL, ActionController::ContentSecurityPolicy, ActionController::RequestForgeryProtection, AbstractController::Callbacks, ActiveSupport::Callbacks, ActionController::FormBuilder, ActionController::Flash, ActionController::Cookies, ActionController::ParameterEncoding, ActionController::StrongParameters, ActiveSupport::Rescuable, ActionController::ImplicitRender, ActionController::BasicImplicitRender, ActionController::MimeResponds, AbstractController::Caching, AbstractController::Caching::ConfigMethods, AbstractController::Caching::Fragments, ActionController::Caching, ActionController::EtagWithFlash, ActionController::EtagWithTemplateDigest, ActionController::ConditionalGet, ActionController::Head, ActionController::Renderers::All, ActionController::Renderers, ActionController::Rendering, ActionView::Layouts, ActionView::Rendering, ActionController::Redirecting, ActiveSupport::Benchmarkable, AbstractController::Logger, ActionController::UrlFor, AbstractController::UrlFor, ActionDispatch::Routing::UrlFor, ActionDispatch::Routing::PolymorphicRoutes, ActionController::Helpers, AbstractController::Helpers, AbstractController::AssetPaths, AbstractController::Translation, AbstractController::Rendering, ActionView::ViewPaths, ActionController::Metal, AbstractController::Base, ActiveSupport::Configurable, ActiveSupport::ToJsonWithActiveSupportEncoder, Object, PP::ObjectMixin, ActiveSupport::Dependencies::Loadable, JSON::Ext::Generator::GeneratorMethods::Object, ActiveSupport::Tryable, Kernel, BasicObject]

**Comment by phuongnl15395** [comment]
@HugoHasenbein any update from my case?

**Comment by phuongnl15395** [comment]
Hi @HugoHasenbein ,
I'm just cloning production instance to new one and remove all plugin except this plugin  "redmine_more_previews". 
After installing + restart I ran into this problem
This fresh without any plugin
![image](https://user-images.githubusercontent.com/67360660/167334566-8715bd42-1cb9-4c0b-8084-bcf75b4e050f.png)
After restart
![image](https://user-images.githubusercontent.com/67360660/167334603-a4a16a5d-d066-4518-b469-d760889b4d9b.png)

**Comment by acosonic** [comment]
I am too having 

```
2.6.3 :001 > AttachmentsController.new.methods.include?(:show_with_more_previews)
 => false
```
And 

` AttachmentsController.ancestors
 => [RedmineMorePreviews::Patches::AttachmentsControllerPatch::ClassMethods, AttachmentsController, RedmineMorePreviews::ControllerHelper, RedmineMorePreviews::Patches::AttachmentsControllerPatch, RedminePreviewInline::Patches::AttachmentsControllerPatch::InstanceMethods, RedminePreviewInline::Patches::AttachmentsControllerPatch, RedmineAllThumbnails::Patches::AttachmentsControllerPatch::InstanceMethods, RedmineAllThumbnails::Patches::AttachmentsControllerPatch, RedmineLightbox2::AttachmentsPatch, #<Module:0x000055914a29f6e0>, ApplicationController, Spectator::Patches::ApplicationControllerInstanceMethods, Spectator::Patches::ApplicationControllerPatch, RedmineExtensions::RenderingHelper, RedmineExtensions::ApplicationControllerPatch::InstanceMethods, RedmineExtensions::ApplicationControllerPatch, Redmine::SudoMode::Controller, Redmine::MenuManager::MenuController, Redmine::Search::Controller, AvatarsHelper, GravatarHelper::PublicMethods, RoutesHelper, Redmine::Hook::Helper, Redmine::Pagination, Redmine::I18n, #<Module:0x0000559148dfb5b8>, #<Module:0x00005591488c1610>, #<Module:0x00005591488c1660>, ActionController::Base, OpenIdAuthentication, InvisibleCaptcha::ControllerExt, ActiveRecord::Railties::ControllerRuntime, ActionDispatch::Routing::RouteSet::MountedHelpers, ActionController::ParamsWrapper, ActionController::Instrumentation, ActionController::Rescue, ActionController::HttpAuthentication::Token::ControllerMethods, ActionController::HttpAuthentication::Digest::ControllerMethods, ActionController::HttpAuthentication::Basic::ControllerMethods, ActionController::DataStreaming, ActionController::Streaming, ActionController::ForceSSL, ActionController::ContentSecurityPolicy, ActionController::RequestForgeryProtection, AbstractController::Callbacks, ActiveSupport::Callbacks, ActionController::FormBuilder, ActionController::Flash, ActionController::Cookies, ActionController::ParameterEncoding, ActionController::StrongParameters, ActiveSupport::Rescuable, ActionController::ImplicitRender, ActionController::BasicImplicitRender, ActionController::MimeResponds, AbstractController::Caching, AbstractController::Caching::ConfigMethods, AbstractController::Caching::Fragments, ActionController::Caching, ActionController::EtagWithFlash, ActionController::EtagWithTemplateDigest, ActionController::ConditionalGet, ActionController::Head, ActionController::Renderers::All, ActionController::Renderers, ActionController::Rendering, ActionView::Layouts, ActionView::Rendering, ActionController::Redirecting, ActiveSupport::Benchmarkable, AbstractController::Logger, ActionController::UrlFor, AbstractController::UrlFor, ActionDispatch::Routing::UrlFor, ActionDispatch::Routing::PolymorphicRoutes, ActionController::Helpers, AbstractController::Helpers, AbstractController::AssetPaths, AbstractController::Translation, AbstractController::Rendering, ActionView::ViewPaths, ActionController::Metal, AbstractController::Base, ActiveSupport::Configurable, ActiveSupport::ToJsonWithActiveSupportEncoder, Object, PP::ObjectMixin, ERB::Util, ActiveSupport::Dependencies::Loadable, JSON::Ext::Generator::GeneratorMethods::Object, ActiveSupport::Tryable, Kernel, BasicObject] 
`

Version Redmine 4.1.0.stable

**Comment by HugoHasenbein** [comment]
Dear Aleksandar,

with version 4.x this test has become obsolete, as from version 4.x on, 
the "prepend"-method is used to patch redmine.
So, the plugin does not work? What ate the symptoms? Can you send a log 
file excerpt?

Best regards

Stephan
Am 2022-05-10 14:47, schrieb Aleksandar Pavić:

**Comment by acosonic** [comment]
If I try to preview xlsx as image I get:

  
ActionController::RoutingError (No route matches [GET] "/tmp/mini_magick20220511-502327-1a96zhk.png"):

If I try to preview as html

then no error, but nothing previewed ...

**Comment by acosonic** [comment]
I can provide you with ssh access to that particular instance...

**Comment by HugoHasenbein** [comment]
that would help :-)

**Comment by ashrafalzyoud** [comment]
@acosonic 
can u give in nginx

**Comment by znkey** [comment]
Try to Open it in project setting page

**Comment by HugoHasenbein** [comment]
Dear Phuong,

I suspect a plugin collision due to the new patching scheme introduced with Rails 5 / redmine 4.

To keep compatibility with both versions, namely redmine < 4.0 and >= 4.0 I circumvented the patching scheme by a generally known alternative to alias_method_chain, which has been deprecated with Rails 5.

Please try attached version 4.0.0.a, which now uses the prepend-method to patch existing classes. If a plugin collision caused the effect, that you report, due to this particular incompatibility, then this version should work.

Unpack the tgz file (the folder should have the name 'redmine_more_previews' not redmine_more_previews-4.0.0a in the plugins folder and tell me how this version behaves.

Best regards

Stephan

**Comment by HugoHasenbein** [comment]
didn't you ttry version 4.0.0a, which hopefully does not interfere with other plugins?

**Comment by HugoHasenbein** [comment]
Dear Phuong,

we'll get that issue sooner or later. Please, try to open a rails console

rails c -e production

It should throw an error message.

Best regards

Stephan

**Comment by HugoHasenbein** [comment]
Dear Phuong,

how does the updated version work?

**Comment by HugoHasenbein** [comment]
Do you have a log file output?

**Comment by phuongnl15395** [comment]
Hi @HugoHasenbein ,

Thanks for not ghosting this, but it's been quite a long time and i'm not using this plugin anymore.

Thanks for your support.

**Comment by chalchal18** [comment]
Hello @HugoHasenbein ,
I also faced this issue. I use Redmine 5.0.1. stable and Rails version is 6.1.6 . Even though I cloned redmine_more_previews to my system, it does not work. Could you please help me to get a solution.I have attached the log file and some screenshot for your reference .
![preview](https://github.com/HugoHasenbein/redmine_more_previews/assets/129147918/2bda9574-617d-4e28-b1d2-31ef273d04ef)
![information](https://github.com/HugoHasenbein/redmine_more_previews/assets/129147918/91ffbdbc-b266-405c-b3c2-b1aeab2b3850)
![log_ss](https://github.com/HugoHasenbein/redmine_more_previews/assets/129147918/49726987-ca25-42ce-8889-abf1e701f967)

**Comment by AntonBuyanov** [comment]
> Привет@HugoHasenbein, я тоже столкнулся с этой проблемой. Я использую Redmine 5.0.1. стабильная версия и версия Rails — 6.1.6. Несмотря на то, что я клонировал redmine_more_previews в свою систему, он не работает. Не могли бы вы помочь мне найти решение. Я приложил файл журнала и скриншот для вашей справки. ![предварительный просмотр](https://private-user-images.githubusercontent.com/129147918/243279988-2bda9574-617d-4e28-b1d2-31ef273d04ef.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4NzczMDEsIm5iZiI6MTcwNjg3NzAwMSwicGF0aCI6Ii8xMjkxNDc5MTgvMjQzMjc5OTg4LTJiZGE5NTc0LTYxN2QtNGUyOC1iMWQyLTMxZWYyNzNkMDRlZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIwMlQxMjMwMDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02NTQ3ZTBmMTdlY2IzODViNGJjNGFhODBmZGEyY2RhMTk5ZWZiZjMyNDMwMGM1YTVlMjk2Mjc3N2U5NTFhMjBmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Tk83BFrw-th2QzOTXyo9ozZ-UmzUoSp0-XGwNGOT6oM) ![информация](https://private-user-images.githubusercontent.com/129147918/243281427-91ffbdbc-b266-405c-b3c2-b1aeab2b3850.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4NzczMDEsIm5iZiI6MTcwNjg3NzAwMSwicGF0aCI6Ii8xMjkxNDc5MTgvMjQzMjgxNDI3LTkxZmZiZGJjLWIyNjYtNDA1Yy1iM2MyLWIxYWVhYjJiMzg1MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIwMlQxMjMwMDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMjY5NmVkZjNkNWE0NTJjNmE0NGQyOTNiMDQ5NWZlNWY1OTMxNWQ5M2ViNGQ5MmEyOTY1NGJlMWI1MzVkZTAwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.QYgT8c0ZYfQaQQ5PNdp28yXxowkeaQ7XEHQ8DHyV68I) ![log_ss](https://private-user-images.githubusercontent.com/129147918/243281614-49726987-ca25-42ce-8889-abf1e701f967.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4NzczMDEsIm5iZiI6MTcwNjg3NzAwMSwicGF0aCI6Ii8xMjkxNDc5MTgvMjQzMjgxNjE0LTQ5NzI2OTg3LWNhMjUtNDJjZS04ODg5LWFiZjFlNzAxZjk2Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIwMlQxMjMwMDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hYTY5Mjc4M2Y4NjlkYmZhNWI0MTAwYmFhMzhlZGQ4MjU4OThjNmRlOTI0NjY4NmVlNzZiNThmNGQyNjk2MWNkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.CebCFP3bh1wtu-x6XwcSFfAQyWbX8pFz30dd0BxojoQ)

Faced the same problem. It turned out that it was necessary to set the checkbox true in the project settings..
![image_2024-02-02_18-31-10](https://github.com/HugoHasenbein/redmine_more_previews/assets/102428399/39e58f4e-3255-4fb3-a696-c6acad8a89e4)
